# Experiment 59905fecaf92ec0cde8ca574f92914fc734484386922525bc25b2a3f38bbca2b
# Base 44d296b30a9b2307b00c4b38f344bc3705dc4219
diff --git a/config/reviewed_reference_identities.json b/config/reviewed_reference_identities.json
index 40c6632ff..674cda77f 100644
--- a/config/reviewed_reference_identities.json
+++ b/config/reviewed_reference_identities.json
@@ -62,6 +62,7 @@
     "0054B192": {"symbol": "_ImmReleaseContext@8", "aliases": []},
     "0054B198": {"symbol": "_ImmNotifyIME@16", "aliases": []},
     "0054B19E": {"symbol": "_ImmGetContext@4", "aliases": []},
+    "0054B1A4": {"symbol": "_ICInfo@12", "aliases": []},
     "0054B1AA": {"symbol": "_MCIWndCreateA", "aliases": []},
     "0054D6E0": {"symbol": "??_L@YGXPAXIHP6EX0@Z1@Z", "aliases": []},
     "0054D780": {"symbol": "??_M@YGXPAXIHP6EX0@Z@Z", "aliases": []},
